examples/five_mem_node/main.rs

```diff
@@ -222,17 +218,7 @@ fn on_ready(
                 continue;
             }
             if let EntryType::EntryConfChange = entry.get_entry_type() {
-                // For conf change messages, make them effective.
-                let mut cc = ConfChange::new();
-                cc.merge_from_bytes(entry.get_data()).unwrap();
-                let node_id = cc.get_node_id();
-                match cc.get_change_type() {
-                    ConfChangeType::AddNode => raft_group.raft.add_node(node_id).unwrap(),
-                    ConfChangeType::RemoveNode => raft_group.raft.remove_node(node_id).unwrap(),
-                    ConfChangeType::AddLearnerNode => raft_group.raft.add_learner(node_id).unwrap(),
-                    ConfChangeType::BeginMembershipChange
-                    | ConfChangeType::FinalizeMembershipChange => unimplemented!(),
-                }
+                // For conf change messages, nothing need to do.
```

Comment:
They won't. Now all configuration changes become effective on received.

Reply:
This changes the semantics of our Add/Remove/AddLearner then. Do we need to prove that it can still work correctly with TLA+? What do you think @BusyJay ?